verticapy/datasets/generators.py

```diff
@@ -173,6 +218,37 @@ def gen_meshgrid(features_ranges: dict) -> vDataFrame:
     -------
     vDataFrame
         generated dataset.
+
+    Examples
+    ---------
+    .. code-block:: python
+
+        from verticapy.datasets import gen_meshgrid
+        import datetime
+
+        gen_meshgrid(features_ranges = {"name": {"type": str, "values": ["Badr", "Badr", "Raghu", "Waqas",]},
+                                        "age": {"type": int, "range": [20, 40]},
+                                        "distance": {"type": float, "range": [1000, 4000]},
+                                        "date": {"type": datetime.date, "range": ["1993-11-03", 365]},
+                                        "datetime": {"type": datetime.datetime, "range": ["1993-11-03", 365]},},)
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import gen_meshgrid
+        import datetime
+        import verticapy as vp
+        html_file = open("datasets_data_generators_gen_meshgrid.html", "w")
+        html_file.write(gen_meshgrid(features_ranges = {"name": {"type": str, "values": ["Badr", "Badr", "Raghu", "Waqas",]},
+                                "age": {"type": int, "range": [20, 40], "nbins": 3,},
+                                "distance": {"type": float, "range": [1000, 4000], "nbins": 3,},
+                                "date": {"type": datetime.date, "range": ["1993-11-03", 365], "nbins": 2,},
+                                "datetime": {"type": datetime.datetime, "range": ["1993-11-03", 365], "nbins": 2,},},)._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: datasets_data_generators_gen_meshgrid.html
```

Comment:
As per the new syntax:
:file: SPHINX_DIRECTORY/figures/datasets_data_generators_gen_meshgrid.html